### ISSUE TITLE
Paragraphs (useLineBreaks) and specific tags (formatBlock)

### ISSUE THREAD

**Issue by zrod** [tiff/wysihtml5#278]
When setting useLineBreaks to false and having a formatBlock with blockquote (for example), each return hit results in a new blockquote tag wrapped in a new p tag.

There should be some sort of filter for specific tags (perhaps let user specify these tags) where a return hit would insert a br tag instead of a new paragraph.

I'll see if I can come up with something.

**Comment by garrytan** [comment]
I agree -- I'd like a new paragraph within that blockquote. Will also investigate too.

**Comment by maduhaime** [comment]
Something similar is happening when inserting HR tag using insertHTML.  The editor will not create P tags after.  It will switch to DIV and BR.
